ansible_base/activitystream/signals.py

```diff
@@ -7,6 +7,20 @@
 logger = logging.getLogger('ansible_base.activitystream.signals')
 
 
+def _always_exclude_fields():
+    """
+    Returns a list of 2-tuples of (model_or_base_class, 'field_name') that
+    should always be excluded from activity stream entries.
```

Comment:
That's just the thing, I think we need these exceptions for more purposes than just activity stream (modified_by for example).